gnd/src/main/java/com/google/android/gnd/ui/home/featuredetails/FeatureDetailsViewModel.java

```diff
@@ -31,16 +30,12 @@
 @SharedViewModel
 public class FeatureDetailsViewModel extends ViewModel {
 
-  public final ObservableField<String> featureTitle;
-  public final ObservableField<String> featureSubtitle;
-  public final ObservableBoolean isFeatureSubtitleVisible = new ObservableBoolean(false);
-
+  public final ObservableField<Optional<Feature>> observableFeature;
```

Comment:
Same comment re "observable" prefix - it can be considered part of how the values are delivered (the how) rather than what is being delivered (the what).